cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/StatefulSetOperator.java

```diff
@@ -27,10 +28,12 @@
 import java.util.function.Predicate;
 
 /**
- * Operations for {@code StatefulSets}s, which supports {@link #rollingUpdate(String, String)}
+ * Operations for {@code StatefulSets}s, which supports {@link #maybeRollingUpdate(StatefulSet)}
  * in addition to the usual operations.
  */
-public class StatefulSetOperator<P> extends AbstractScalableResourceOperator<KubernetesClient, StatefulSet, StatefulSetList, DoneableStatefulSet, RollableScalableResource<StatefulSet, DoneableStatefulSet>, P> {
+public abstract class StatefulSetOperator extends AbstractScalableResourceOperator<KubernetesClient, StatefulSet, StatefulSetList, DoneableStatefulSet, RollableScalableResource<StatefulSet, DoneableStatefulSet>> {
+
+    public static final String ANNOTATION_GENERATION = "cluster-controller.strimzi.io/statefulset-generation";
```

Comment:
why not using the `STRIMZI_DOMAIN` (just `strimzi.io`) we have in the `Labels` class?

Reply:
Actually shouldn't we use `ClusterOperator.STRIMZI_CLUSTER_OPERATOR_DOMAIN`, which is what we use for the delete claim annotation?

Reply:
You are right but now I'm thinking ... does it make sense this specific cluster operator domain or the strimzi domain is enough. But up to you ... no strong opinion on that from me.